### ISSUE TITLE
Error while running under npm v6.11 in Windows

### ISSUE THREAD

**Issue by swayams** [istanbuljs/nyc#1163]
## Issue repository
https://github.com/swayams/geektrust-solutions

## Expected Behavior
I have setup nyc + babel following https://istanbul.js.org/docs/tutorials/es2015/ when I run test task it should show me passed tests alogn with coverage. If i remove nyc from task command ( ie just run mocha, it works fine )

## Actual Behavior 
When I use nyc along with mocha and babel, I get error that node module is not found. I have cleaned and reinstalled. Issue persists. Please refer below for error details.

## System Information

  **System:**
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Memory: 7.78 GB / 15.88 GB
  **Binaries:**
    Node: 8.12.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.11.1 - C:\Program Files\nodejs\npm.CMD
   **npmPackages:**
    @babel/cli: ^7.5.5 => 7.5.5
    @babel/core: ^7.5.5 => 7.5.5
    @babel/node: ^7.5.5 => 7.5.5
    @babel/plugin-proposal-class-properties: ^7.5.5 => 7.5.5
    @babel/preset-env: ^7.5.5 => 7.5.5
    @babel/register: ^7.5.5 => 7.5.5
    babel-node: 0.0.1-security => 0.0.1-security
    babel-plugin-istanbul: ^5.2.0 => 5.2.0
    nyc: ^14.1.1 => 14.1.1

# Output
```
 module.js:550
    throw err;
    ^

Error: Cannot find module 'C:\Users\siddhswa\study\geektrust-solutions\Geektrust-problem-1\node'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3


``` module.js:550
    throw err;
    ^

Error: Cannot find module 'C:\Users\siddhswa\study\geektrust-solutions\Geektrust-problem-1\node'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3`

**Comment by coreyfarrell** [comment]
This issue has been caused by changes in npm v6.11 and is Windows specific.  I've updated the title of this issue so other users can find it more easily.  The proposed fix is at https://github.com/istanbuljs/spawn-wrap/pull/102, I'm hoping to get that merged soon but the changes involve some pretty advanced (or obscure?) Windows scripting so I'm working on getting a review from someone with appropriate knowledge.

**Comment by coreyfarrell** [comment]
Please run `npm update --depth 9999` to update your package-lock.json then try again.  An updated version of spawn-wrap was just released which should resolve this issue.

**Comment by swayams** [comment]
Hi @coreyfarrell ,  `npm update --depth 9999` didn't help. The `spawn-wrap` package has the same version before and after the change. The version referenced in my project is 1.4.2. Am I missing something?

**Comment by swayams** [comment]
Hi @coreyfarrell , I followed your instructions and it worked out fine. Thanks for everything. Much appreciated.

**Comment by coreyfarrell** [comment]
For anyone else who runs across this issue it looks like the `npm` bug with not honoring `--depth` is fixed in v6.11.3 - https://github.com/npm/cli/releases/tag/v6.11.3.
